lib/MiniDroneBtAdapter.js

```diff
@@ -50,8 +58,18 @@ const FLIGHT_STATUSES = ['landed', 'taking off', 'hovering', 'flying',
  * @author Christopher Fetherston <chris@cfetherston.com>
  */
 class MiniDroneBtAdapter extends EventEmitter {
-    constructor() {
+    /**
+     * Instantiates a new instance of the MiniDroneBtAdapter class
+     *
+     * @param {Object} options Configuration options object
+     * @param {String} options.droneFilter The name of the drone to restrict connection to
+     */
+    constructor(options) {
         super();
+        const defaults = {
+            droneFilter: '',
+        };
+        this.options = Object.assign({}, defaults, options);
```

Comment:
I'm not very familliar with mocha yet so this may not be the best way to do it but test work fine.
https://github.com/gotleg/npm-parrot-minidrone/commit/ca85ef41dca5a80ffd58b9b9f92f4954ef98f1ec